ios/native/UIResponder+Reanimated.mm

```diff
@@ -7,12 +7,19 @@
 #import <React/RCTCxxBridgeDelegate.h>
 #import <RNReanimated/REAEventDispatcher.h>
 
-#if RNVERSION >= 64
```

Comment:
Why do we need this change. Isn't RNVERSION clearer?

Reply:
I think by checking if the JS-CallInvoker file can be included we stay future proof - just in case anything changes in the future which requires some additional RNVERSION checks that get really complicated. It takes me longer to read that `RNVERSION >= ...` or `RNVERSION < ...` than just `if has include` (which sounds more like proper english)

Reply:
The current approach is more informative in my opinion. This informs in a clear way why is possible these header files can don't exist, and which header file is related to which the react-native version.

Reply:
ai ai captain, will revert in a few mins

Reply:
reverted those changes. 👍 @piaskowyk @Szymon20000